UnicodeDecodeError with Python3

When following the build instructions, I got an error when issuing the 'ninja' build command. This happend on a fresh clone of the repo:
```
git clone https://github.com/google/shaderc.git
cd shaderc
./utils/git-sync-deps
git checkout known-good
./update_shaderc_sources.py
git checkout master
mkdir build && cd build
cmake -GNinja -DCMAKE_BUILD_TYPE=Release ..
ninja

[6/756] Update build-version.inc in the Shaderc build directory (if necessary).
FAILED: CMakeFiles/build-version 
cd /home/danlar02/WorkDir/shaderc/build && /usr/bin/python3 /home/danlar02/WorkDir/shaderc/utils/update_build_version.py /home/danlar02/WorkDir/shaderc /home/danlar02/WorkDir/shaderc/third_party/spirv-tools /home/danlar02/WorkDir/shaderc/third_party/glslang
Traceback (most recent call last):
  File "/home/danlar02/WorkDir/shaderc/utils/update_build_version.py", line 137, in <module>
    main()
  File "/home/danlar02/WorkDir/shaderc/utils/update_build_version.py", line 125, in main
    for (p, d) in zip(projects, sys.argv[1:])
  File "/home/danlar02/WorkDir/shaderc/utils/update_build_version.py", line 125, in <listcomp>
    for (p, d) in zip(projects, sys.argv[1:])
  File "/home/danlar02/WorkDir/shaderc/utils/update_build_version.py", line 111, in get_version_string
    detailed_version_string_lst.append(deduce_software_version(directory))
  File "/home/danlar02/WorkDir/shaderc/utils/update_build_version.py", line 67, in deduce_software_version
    for line in f.readlines():
  File "/usr/lib/python3.6/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 826: ordinal not in range(128)
[23/756] Building CXX object third_party/googletest/googletest/CMakeFiles/gtest.dir/src/gtest-all.cc.o
ninja: build stopped: subcommand failed.
```

Changing all 'python3' commands to use 'python2' in build/build.ninja fixed the issue, but seems there's something wrong?

I'm running Python 3.6.8 on Ubuntu 18.04.2

--- a/utils/update_build_version.py
+++ b/utils/update_build_version.py
@@ -63,7 +63,7 @@ def deduce_software_version(directory):
     # Linux.
     pattern = re.compile(r'^(v\d+\.\d+(-dev)?) \d\d\d\d-\d\d-\d\d\s*$')
     changes_file = os.path.join(directory, 'CHANGES')
-    with open(changes_file) as f:
+    with open(changes_file, errors='replace') as f:
         for line in f.readlines():
             match = pattern.match(line)
             if match:
@@ -115,8 +115,8 @@ def get_version_string(project, directory):
 
 def main():
     if len(sys.argv) != 4:
-        print('usage: {} <shaderc-dir> <spirv-tools-dir> <glslang-dir>'.format(
-            sys.argv[0]))
+        print(('usage: {} <shaderc-dir> <spirv-tools-dir> <glslang-dir>'.format(
+            sys.argv[0])))
         sys.exit(1)
 
     projects = ['shaderc', 'spirv-tools', 'glslang']

